src/main/kotlin/minesweeper/MineSweeperMain.kt

```diff
@@ -0,0 +1,27 @@
+package minesweeper
+
+import minesweeper.view.InputView
+import minesweeper.view.OutputView
+import kotlin.random.Random
+
+fun main() {
+    val height = InputView.askHeight()
+    val width = InputView.askWidth()
+    val mineCount = InputView.askMineCount(height * width)
+
+    val mineCells = initMineCells(height, width, mineCount)
+    val mineBoard = MineBoard(mineCells)
+    OutputView.printMineBoard(mineBoard)
+}
+
+private fun initMineCells(height: Int, width: Int, mineCount: Int): Array<Array<MineCell>> {
```

Comment:
```suggestion
private fun initMineCells(height: Int, width: Int, mineCount: Int): Array<MineBoardRow> {
```

생성한 일급컬렉션을 사용해보는건 어떨까요?